oak_runtime/src/node/grpc/client.rs

```diff
@@ -85,6 +85,8 @@ impl GrpcClientNode {
         // We compute the node privilege once and for all at start and just store it, since it does
         // not change throughout the node execution.
         let node_privilege = grpc_client_node_privilege(&uri);
+        let root_tls_certificate = crate::config::load_certificate(&root_tls_certificate_bytes)
+            .map_err(|_err| ConfigurationError::InvalidNodeConfiguration)?;
```

Comment:
Also log the error?

Reply:
I don't think that is needed. The error (`_err`) does not have any specific content (its type is`()`), and `load_certificate` already logs an error.